docs/interpreter/mongodb.md

```diff
@@ -45,7 +45,7 @@ Second, create mongodb interpreter in Zeppelin.
   </tr>
   <tr>
     <td>mongo.shell.path</td>
-    <td>mongo</td>
+    <td>mongosh</td>
     <td>MongoDB shell local path. <br/> Use `which mongo` to get local path in linux or mac.</td>
```

Comment:
the description should be updated too. also be good to mention `mongosh`'s applicable versions

Reply:
@pan3793 Thank you. I updated description too. [77a39e1](https://github.com/apache/zeppelin/pull/4846/commits/77a39e105a02d104b769f4d31480b74400808439)